src/test/integration/users/User.sol

```diff
@@ -0,0 +1,261 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "forge-std/Test.sol";
+
+import "src/contracts/core/DelegationManager.sol";
+import "src/contracts/core/StrategyManager.sol";
+import "src/contracts/pods/EigenPodManager.sol";
+
+import "src/contracts/interfaces/IDelegationManager.sol";
+import "src/contracts/interfaces/IStrategy.sol";
+
+import "src/test/integration/TimeMachine.t.sol";
+
+contract User is Test {
+
+    Vm cheats = Vm(HEVM_ADDRESS);
+
+    DelegationManager delegationManager;
+    StrategyManager strategyManager;
+    EigenPodManager eigenPodManager;
+
+    IStrategy[] depositedStrategies;
```

Comment:
Do we need these? Top-level it's been easy to get by without them and I worry that too much stuff like this starts to accumulate logic in the User contracts

Reply:
by these i mean `depositedStrategies`

Reply:
[c99e847](https://github.com/Layr-Labs/eigenlayer-contracts/pull/335/commits/c99e847709852d7246c73b7d72d44bba368b760e)